packages/merkle-distributor/src/merkleTree.js

```diff
@@ -0,0 +1,131 @@
+const web3 = require("web3");
+const ethereumjsUtil = require("ethereumjs-util");
+
+class MerkleTree {
```

Comment:
Just curious -- is this code inspired or pulled from somewhere else?

Reply:
see above ^^ https://github.com/UMAprotocol/protocol/pull/2598#discussion_r586316304